solpolpy/core.py

```diff
@@ -66,6 +71,11 @@ def resolve(input_data: t.Union[t.List[str], NDCollection], out_system: str) ->
     equation = get_transform_equation(transform_path)
     requires_alpha = check_alpha_requirement(transform_path)
 
+    # TODO - Does this apply for all transformations, or only within MZP->MZP correction?
+    # TODO - If so, should this result in an error when trying to use it outside of this scope?
+    if imax_effect:
```

Comment:
@s0larish should comment on this. I would like to see an error as you propose if it's specific to MZP->MZP.

Reply:
Yes it's specific to MZP -> MZP. We can make an estimate if this correction is not done on WFI what percentage of error in pB we can get.